airbyte-integrations/connectors/source-holiday-api/source_holiday_api/manifest.yaml

```diff
@@ -0,0 +1,343 @@
+version: 0.79.0
+type: DeclarativeSource
+check:
+  type: CheckStream
+  stream_names:
+    - holidays
+    - countries
```

Comment:
You don't have to have multiple streams in check — just one is generally fine, especially for a simple API like this.